kdeploy

```diff
@@ -83,6 +94,12 @@ usage() {
 	    --wait-remote-reboot	Like --remote-reboot (implied), but also wait it
 	                                come back [ default: no ]
 	    -t, --tmpdir[=] DIR		Directory to use for temporary installation
+	    --remote-tmpdir[=] DIR	Set the remote directory to use as temporary location
+	                                for installation. Set it to an empty string or "no" to
+	                                disable the use of a temporary location. It requires remote
```

Comment:
I would prefer `--no-remote-tmpdir` instead. Your call though.

Reply:
makes sense. I will add the --no-remote-tmpdir, remove the --remote-tmpdir=no, but keep the behavior for --remote-tmpdir=

Reply:
thanks for checking

Reply:
Hm... For clarity: I checked the code, but haven't tested it. Would you like me to test it as well?

Reply:
testing is good, but no pressure... this is more a toy helper than a production-ready thing